### PR TITLE
ed/uu updates

### PR DESCRIPTION
**Pull request by urcades**

### CODE REVIEW

File: content/understanding-urbit/interface.md [hunk 1]
```diff
@@ -8,56 +8,64 @@ hide_next_title = "true"
...
+<img class="ba" src="https://storage.googleapis.com/media.urbit.org/site/understanding-urbit/uu-interface-3.png">
 
-Creating a group and customizing the modules it shares is easy. Building a new module is pretty straightforward as well, no more difficult than building an iOS app — although you do have to learn about the Urbit stack.
+We built OS 1 for ourselves. We were sick of using centralized, mainstream software. We didn’t want to be stuck switching between bloated, monolithic services in order to keep in touch and move our work forward. We were done relying on some third party to take care of our community—so we exited to Urbit, and it feels good.
 
-<img class="full w-100 mv4" src="https://media.urbit.org/site/understanding-urbit/uu-osn-3.svg"/>
+If you’re curious to get a complete overview of OS 1, check out [this post](https://urbit.org/blog/introducing-os1/), watch the [release event](https://www.youtube.com/watch?v=71ViyftPkGk&feature=youtu.be&t=3963) or feel free to [boot a node](http://127.0.0.1:1111/using/install/#comet) to come talk to us.
```
Comment (matildepark) [inline content/understanding-urbit/interface.md]:
```suggestion
If you’re curious to get a complete overview of OS 1, check out [this post](https://urbit.org/blog/introducing-os1/), watch the [release event](https://www.youtube.com/watch?v=71ViyftPkGk&feature=youtu.be&t=3963) or feel free to [boot a node](@/using/install.md#comet) to come talk to us. 
```

File: content/understanding-urbit/roadmap.md [hunk 2]
```diff
@@ -8,73 +8,91 @@ hide_next_title = "true"
...
 
+We started out by building a prototype for what this future foundation should look like. We spent the early years working primarily on the architecture of the system—but a new platform needs a clear use-case, a way to put it to work. This is what we’ve been building in the last year or so. It’s called Landscape.
 
-![](https://media.urbit.org/site/understanding-urbit/project-history/project-status-landscape%402x.png)
+Landscape is a flexible tool for communicating and collaborating with friends that runs on top of Urbit OS. 
 
-**OS 0 — 2019**
+Landscape lets you bring a group of people together to communicate, collaborate, and do business. Think of it as a combination of ‘productivity software’ and ‘social networking’. Landscape is for bringing together a group of people to share a set of ‘modules’, or things to do, and customizing an environment for the group to stay connected. For a full rundown, see the [interface section](/interface).
```
Comment (matildepark) [inline content/understanding-urbit/roadmap.md]:
```suggestion
Landscape lets you bring a group of people together to communicate, collaborate, and do business. Think of it as a combination of ‘productivity software’ and ‘social networking’. Landscape is for bringing together a group of people to share a set of ‘modules’, or things to do, and customizing an environment for the group to stay connected. For a full rundown, see the [interface section](https://urbit.org/understanding-urbit/interface/).
```

File: content/understanding-urbit/roadmap.md [hunk 3]
```diff
@@ -8,73 +8,91 @@ hide_next_title = "true"
...
+### Today: Landscape OS 1.N
 
-OS 1 is aimed first and foremost at communities looking to homestead a new platform they can truly make their own. We’ve been using Urbit ourselves over the years, and OS 1 is what we’ve been looking forward to using for our own daily communication and collaboration.
+We built OS 1 for ourselves, because we wanted to use a communication tool that we can trust and put our system to work for real. 
 
-While we’re excited about what OS 1 is, there are also plenty of gaps. For example, OS 1 doesn’t have an exceptional notification system, it still lives in a browser, and it lacks great tooling for distributing new modules. While we aren’t inventing any new cryptography, Urbit also hasn’t been audited, so we aren’t calling it secure just yet.
+Today we use OS 1 every day to chat, write, and share links. It’s a calm, minimalist tool that gets us one step away from the ads, tracking, distraction and disruption of mainstream software. (We won’t get into the details of OS 1 here, but you can check out [this post](https://urbit.org/blog/introducing-os1/) for a walkthrough. You’re also welcome to try [booting a node yourself](https://urbit.org/using/install/))
```
Comment (matildepark) [inline content/understanding-urbit/roadmap.md]:
```suggestion
Today we use OS 1 every day to chat, write, and share links. It’s a calm, minimalist tool that gets us one step away from the ads, tracking, distraction and disruption of mainstream software. (We won’t get into the details of OS 1 here, but you can check out [this post](https://urbit.org/blog/introducing-os1/) for a walkthrough. You’re also welcome to try [booting a node yourself](https://urbit.org/using/install/).)
```

File: content/understanding-urbit/interface.md (hunk 1 above)
Comment (urcades) [reply content/understanding-urbit/interface.md]: damn, good catch

**Comment by matildepark** [comment]
Bombs away!

**Pull request merged**